### DISCUSSION TITLE
Error during installation HyperHDR

### DISCUSSION THREAD

**Discussion by newcgen** [awawa-dev/HyperHDR#141] [Q&A] [answered]
Hi!

I am successfully running Hyper HDR on Khadas VIM3. The installation does not work.
Ubuntu 20.04

`sudo make install/strip`

CMake Error at dependencies/external/libjpeg-turbo/cmake_install.cmake:95 (file):
  file INSTALL cannot find
  "/home/khadas/work/hyperhdr/build/dependencies/external/libjpeg-turbo/pkgscripts/libjpeg.pc":
  No such file or directory.
Call Stack (most recent call first):
  dependencies/cmake_install.cmake:43 (include)
  cmake_install.cmake:42 (include)

**Comment by awawa-dev** [comment, marked as answer]
Hi
I'll take a look. Usually it's not necessary to use make install: for a testing purpose hyperhdr can be run directly from the build folder and for a permanent and safer installation is recommended to build the installer (````cpack```` command inside the build folder, more in wiki) and then proceed with a package installation.
Regards
Awawa

**Reply by newcgen** [reply]
Cool, all is working. Thanx!
